loki/bulk/scheduler.py

```diff
@@ -576,7 +590,9 @@ def write_cmake_plan(self, filepath, mode, buildpath, rootpath):
         sources_to_remove = []
         sources_to_transform = []
 
-        for item in self.items:
+        # Filter the SGraph to get a pure call-tree
+        item_filter = None if self.config.enable_imports else ProcedureItem
+        for item in SFilter(self.sgraph, item_filter=item_filter):
```

Comment:
Thinking loud here, but could we not do this directly via the filegraph? After all, we're after files here... That would automatically apply the filtering.

Reply:
Ah, yes, in that case leave as is and maybe flag this in an issue? Probably more generally the concept of propagating item properties to file items. Just out of curiosity, how is that property specified in the first place?

Reply:
Filed #255 to track this